bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/i18n/luxtronikheatpump_de.properties

```diff
@@ -0,0 +1,399 @@
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.4 = Sperrzeit
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.5 = Brauchwasser
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.6 = Info Ausheizprogramm
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.7 = Abtauen
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.8 = Pumpenvorlauf
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.9 = Thermische Desinfektion
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.10 = Kühlbetrieb
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.12 = Schwimmbad / Photovoltaik
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.13 = Heizen ext. Energiequelle
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.14 = Brauchwasser ext. Energiequelle
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.16 = Durchflussüberachung
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeile3.state.option.17 = Zweiter Wärmeerzeuger 1 Betrieb
+channel-type.luxtronikheatpump.HauptMenuStatus_Zeit.label = Status Zeit Zeile 2
+channel-type.luxtronikheatpump.HauptMenuAHP_Stufe.label = Stufe Ausheizprogramm
+channel-type.luxtronikheatpump.HauptMenuAHP_Temp.label = Temperatur Ausheizprogramm
+channel-type.luxtronikheatpump.HauptMenuAHP_Zeit.label = Laufzeit Ausheizprogramm
+channel-type.luxtronikheatpump.SH_BWW.label = Brauchwasser aktiv/inaktiv Symbol
+channel-type.luxtronikheatpump.SH_HZ.label = Heizung Symbol
+channel-type.luxtronikheatpump.SH_MK1.label = Mischkreis 1 Symbol
+channel-type.luxtronikheatpump.SH_MK2.label = Mischkreis 2 Symbol
+channel-type.luxtronikheatpump.Einst_Kurzrpgramm.label = Einstellung Kurzprogramm
+channel-type.luxtronikheatpump.StatusSlave1.label = Status Slave 1
+channel-type.luxtronikheatpump.StatusSlave2.label = Status Slave 2
+channel-type.luxtronikheatpump.StatusSlave3.label = Status Slave 3
+channel-type.luxtronikheatpump.StatusSlave4.label = Status Slave 4
+channel-type.luxtronikheatpump.StatusSlave5.label = Status Slave 5
+channel-type.luxtronikheatpump.AktuelleTimeStamp.label = Aktuelle Zeit der Wärmepumpe
+channel-type.luxtronikheatpump.SH_MK3.label = Mischkreis 3 Symbol
+channel-type.luxtronikheatpump.Sollwert_TVL_MK3.label = Mischkreis 3 Vorlauf-Soll-Temperatur
+channel-type.luxtronikheatpump.Temperatur_TFB3.label = Mischkreis 3 Vorlauftemperatur
+channel-type.luxtronikheatpump.MZ3out.label = Ausgang "Mischkreis 3 Zu"
+channel-type.luxtronikheatpump.MA3out.label = Ausgang "Mischkreis 3 Auf"
+channel-type.luxtronikheatpump.FP3out.label = Pumpe Mischkreis 3
+channel-type.luxtronikheatpump.Time_AbtIn.label = Zeit bis Abtauen
+channel-type.luxtronikheatpump.Temperatur_RFV2.label = Raumtemperatur Raumstation 2
+channel-type.luxtronikheatpump.Temperatur_RFV3.label = Raumtemperatur Raumstation 3
+channel-type.luxtronikheatpump.SH_SW.label = Schaltuhr Schwimmbad Symbol
```

Comment:
```suggestion
channel-type.luxtronikheatpump.SH_SW.label = Schaltuhr Schwimmbadsymbol
```

Reply:
guess that would change the meaning. Maybe `Schaltuhr-Schwimmbad-Symbol` would be better?

Reply:
Sounds good!